frontend/cypress/integration/common/navigation.ts

```diff
@@ -2,5 +2,5 @@ import { Given } from 'cypress-cucumber-preprocessor/steps';
 
 Given('user is at the {string} page', (page: string) => {
   // Forcing "Pause" to not cause unhandled promises from the browser when cypress is testing
-  cy.visit(Cypress.config('baseUrl') + `/console/${page}?refresh=0`);
+  cy.visit(Cypress.config('baseUrl') + `console/${page}?refresh=0`);
```

Comment:
Tests fail for me when run locally with the defaults from `make cypress-ui`

Reply:
Hmm, I see the problem, I started cypress with `make -e CYPRESS_BASE_URL=http://localhost:3000/ cypress-gui`.  My trailing slash caused the problem.  I'll revert the change.